alerts/kafka-gke/README.md

```diff
@@ -0,0 +1,33 @@
+# Kafka Alerts for GKE
+
+## Under replicated partitions alert
+When there is an under replicated partition for an extended period of time, then the cluster is not healthy and cannot guarantee high availability.
+
+## Change in number of ISRs alert
+In sync replicas (ISRs) are required for proper failover. The number of in sync replicas should be static, unless the broker cluster is expanding or partitions are removed. If the number of in sync replicas deviates without the broker cluster being modified, you should investigate to maintain high availability.
```

Comment:
Is  "In Sync Replicas" a term? If so, how about we capitalize it here and in the descriptions below?